Dependencies/babylon_native_utils/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+set(SOURCES
+    "Include/Babylon/NetworkUtils.h"
+    "Include/Babylon/ticketed_collection.h"
+    "Source/NetworkUtils.cpp")
+
+add_library(BabylonNativeUtils ${SOURCES})
+
+target_include_directories(BabylonNativeUtils PRIVATE "Include/Babylon")
```

Comment:
Ugh, this naming scheme is killing me. The directory name is snake case but the library is upper camel case?